pkg/webhook/elasticquota/quota_info.go

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package elasticquota
+
+import (
+	v1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/scheduler-plugins/pkg/apis/scheduling/v1alpha1"
+
+	"github.com/koordinator-sh/koordinator/apis/extension"
+)
+
+type QuotaInfo struct {
```

Comment:
we define a new struct, and if you want to add fields in the future, do you need to modify both places at the same time?

Reply:
There are two reasons to define a new struct, first remove the dependency on scheduler; second, webhook no need to check some members of the quotaInfo, such as Used, Runtime, Request and so on. To this question, if the fields we want to add need to be checked in webhook, then we should add the fields both places.